src/components/common/cockpit-components-form-helper.jsx

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
I'd rather import this from pkg/lib, and land https://github.com/cockpit-project/cockpit/pull/18641 first, to avoid duplication.